### DISCUSSION TITLE
Business Idea: Marketing Agency

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#987] [General]
So I have been thinking over the last couple of weeks about what kind of business I would find interesting to start. Since I've been interested in web design and website development, I thought starting a marketing agency could be a good idea.

I would initially start with web development to build a customer base, and eventually expand to other revenue section once I would have enough profit and could start hiring employees. I could grow by adding social media marketing, advertising, and content creation, once I have built a strong foundation. I know this is not very detailed yet and I might be thinking a little far ahead but I believe setting long term goals helps to increase motivation, determination, and productivity.

Let me know if you guys have any sort of tips or recommendations. Just trying to expand my knowledge as I'm new to this and would love to have some feedback from people on this forum.

**Comment by hernessteve** [comment]
I am no marketing agency opening consultant, but I would advise just one thing - skew value.

As a small business owner I`ve entertained the idea of hiring such an agency and after several attempts of contacting people claiming to be such agents I dropped the idea. Why? Because none of them succeeded in answering one simple question: What will you give me for my money? Their answers were: web page, posts, visibility, clicks. But I do not need those. I need sales. And none was able to estimate my gains over expenditures.

In the "Invent with me" podcast hosts mentioned a type of agent I would happily pay to when it is relevant. They call him "an Amazon guy/gal". Such person takes charge of your Amazon sales pages, directs you to produce quality pages, thumbnails, descriptions etc and gets PERCENT of all Amazon sales which he/she is responsible to grow. That is a fair deal for a business owner, I like it.

**Comment by MirkosMangle** [comment]
Good luck with that! I hope the new approach works out well for you. Establishing a solid marketing agency could indeed be a lucrative venture, depending on how you promote it. Utilizing innovative tools like [https://clerk.chat/industries/political/](https://clerk.chat/industries/political/) is crucial in this industry. They can help streamline communication and outreach efforts, giving you a competitive edge and increasing the effectiveness of your campaigns.
